cloud-run/fhir-converter/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM python:3.10
+
+# Install dotnet
+RUN curl -sSL https://dot.net/v1/dotnet-install.sh | \
+    bash /dev/stdin -Channel 3.1 -InstallDir /usr/share/dotnet && \
+    ln -s /usr/share/dotnet/dotnet /usr/bin/dotnet
+
+# Download FHIR-Converter
+WORKDIR /build
+RUN git clone https://github.com/microsoft/FHIR-Converter.git --branch v5.0.4 --single-branch
+WORKDIR /build/FHIR-Converter/src/Microsoft.Health.Fhir.Liquid.Converter.Tool
+
+# Build Microsoft FHIR Converter binary. Need to run twice due to strange error copying oras.exe
+RUN dotnet build || true
+RUN dotnet build
+
+WORKDIR /app
+
+COPY requirements.txt .
+RUN pip install -r requirements.txt
+
+COPY main.py .
+
+EXPOSE 8000
+CMD uvicorn main:api --host 0.0.0.0 --port 8000
```

Comment:
I believe the GCP Clound run default is to use port 8080. Could we switch to 8080?